iib/workers/tasks/build.py

```diff
@@ -778,6 +779,44 @@ def _verify_index_image(
         )
 
 
+def inspect_related_images(bundles: List[str], request_id) -> None:
+    """
+    Verify if related_images in the bundle are pullable.
+
+    :param list bundles: a list of strings representing the pull specifications of the bundles to
+        add to the index image being built.
+    :param int request_id: the ID of the request this index image is for.
+    :raises IIBError: if one of the bundles does not have the pullable related_image.
+    """
+    from iib.workers.tasks.build_regenerate_bundle import _get_bundle_metadata
```

Comment:
It looks like we are importing a private function here. We do have it in other places too so I won't block the merging of the PR on this but if it's not too much work, could you please move that function somewhere else?  If it is, please feel free to file a separate issue for it and resolve the comment

Reply:
I will add a separate PR/commit for fthat

Reply:
created https://github.com/release-engineering/iib/pull/560